whole-tale/gwvolman
# Use new updated time to detect workspace modifications

Fixes #121 

**Approach**:
Use the new folder["updated"] field introduced by https://github.com/whole-tale/virtual_resources/pull/10 in place of tale[workspaceModified] to detect workspace changes during build.

**Prerequisites**:
* https://github.com/whole-tale/girder_wholetale/pull/433
* https://github.com/whole-tale/virtual_resources/pull/10
* `docker pull gwvolman/latest` (see https://github.com/whole-tale/gwvolman/pull/122)

**Test steps**:
* Create a JupyterLab tale
* Run the tale
* Create `apt.txt` with contents `vim`
* Rebuild the tale
* The image should rebuild
